### PR TITLE
[bug 1141019] [bug 1141020] [bug 1141022] User authentication pages

### PR DESCRIPTION
**Pull request by rehandalal**
r?

### CODE REVIEW

File: app/authentication.html [hunk 1]
```diff
@@ -24,36 +27,31 @@
   <body class="skin-organic">
   <section role="region" class="vbox fit">
     <header>
-      <button id="close" data-icon="close" data-action="back">
+      <a href="question_list_helpee.html" id="close" data-icon="close" data-action="back">
```
Comment (Rik) [inline app/authentication.html]: Can you make that `index.html`? It's not use in practice but it's useful in development.

File: app/css/error-list.css [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+    color: #e51e1e;
+    font-size: 14px;
+}
+.ErrorList > li {
+    margin-bottom: 6px;
+}
+.ErrorList > li:last-of-type {
+    margin-bottom: 10px;
+}
+input + .ErrorList > li {
```
Comment (Rik) [inline app/css/error-list.css]: This is too complex of a selector. I'm having a hard time understanding how those margins work. I suggest using more meaningful class names and setting those inside `Utils.refresh_error_list` or in the HTML of the page.

Some name suggestions for the concepts we're trying to style: FormErrorList, FieldErrorList.

File: app/css/authentication.css [hunk 3]
```diff
@@ -0,0 +1,26 @@
+.AuthForm {
+    padding: 15px;
+}
+.AuthForm > p {
```
Comment (Rik) [inline app/css/authentication.css]: Can you make that `.AuthForm-description` and set the appropriate classes where needed.

File: app/css/authentication.css [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+    width: 100%;
+}
+.gaia-footer .bb-button {
+    margin: 0;
+}
+.ForgotPasswordLink {
+    color: #00caf2;
+    font-size: 11px;
+}
+.ErrorList {
```
Comment (Rik) [inline app/css/authentication.css]: Why is this in this file and not `error-list.css`?

Comment (rehandalal) [inline app/css/authentication.css]: i was overriding it for these pages. but it should probably have a `--auth` and then be applied to the relevant places?

Comment (Rik) [inline app/css/authentication.css]: Exactly :)

**Pull request merged**